app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
The function `go_to_cafe` could use some type hinting for its return type. It is a good practice to use type hinting as it makes the code easier to understand and debug. Consider adding a type hint to this function. (Code Style issue)